src/lib/styles/global/utility-classes.scss

```diff
@@ -44,6 +44,11 @@
 
   padding: var(--padding-2x);
   border-radius: var(--border-radius);
+
+  &--highlighted {
+    background: var(--card-background-disabled);
```

Comment:
can we interfer another colod? usigin "disabled" for "highlighted" is a bit suprising without context I guess

Reply:
Good point, was replaced w/ `var(--island-card-background);`

Reply:
Cool 👍